src/map.cpp

```diff
@@ -285,6 +285,21 @@ struct TimeOrderedMapBlock {
 	};
 };
 
+struct DistanceOrderedMapBlock {
```

Comment:
priority queues let you specify a custom comparator, there is no need to have such a "distance ordered mapblock wrapper" as far as I can tell